Exercise_C/index.css

```diff
@@ -2,3 +2,23 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+.flexcontainer {
+  display: flex;
+  flex-flow: row nowrap;
+  height: 200px;
+  justify-content: space-between;
+}
+.blocks {
+  height: 100px;
+  width: 100px;
+}
+.blue {
+  background-color: blue;
+}
+.green {
+  background-color: green;
+}
+.red {
+  background-color: red;
+  align-self: flex-end;
+}
```

Comment:
That's an excellent use of flex. This shows that you have a decent understanding of how flex works